src/psycopt2d/train_model.py

```diff
@@ -126,7 +133,6 @@ def stratified_cross_validation(
 @hydra.main(
     config_path=CONFIG_PATH,
     config_name="default_config",
-    version_base="1.2",
```

Comment:
```suggestion
    version_base="1.2",
```

Deleting this causes warnings, we probably want to have a minimum version of hydra. Any reason for deleting? :-)

Reply:
It is not deleted